runtime_tests/test_samv71_runtime.py

```diff
@@ -124,3 +124,146 @@ def test_samv71_ada_og(taste_project):
     # Check expected compilation output
     stderr = build.stderr.decode('utf-8')
     assert build.returncode == 0, 'Compilation errors: \n{}'.format(stderr)
+
+##
+# TASTE SAMV71 runtime supports deployment of TASTE models containing at least 8 TASTE functions (including
+# at least 4 EDS based). This is validated by the demonstration application (DemoSat2) which contains 14 EDS functions
+# and 1 Linux functions.
+# \SRS  ETB-PER-10
+@pytest.mark.parametrize('taste_project',
+                         ['Demo-Sat-2/src/DemoSat2'],
+                         indirect=True)
+def test_TasteSamV71RuntimeSupportsAtLeastEightFunctions(taste_project):
+    build = common.do_build(taste_project, ['hw', 'debug'])
+
+    # Check expected compilation output
+    stderr = build.stderr.decode('utf-8')
+    assert build.returncode == 0, 'Compilation errors: \n{}'.format(stderr)
+    pass
+
+##
+# Date: 2022.12.09
+# Result: Hwas XML is included in chapter 5.1.4.1 of ETB-N7S-ADD-001.
+# \SRS  ETB-IF-220
+def test_HwasProvidedAsSeds():
+    pass
+
+##
+# As per chapter 5.1 of ETB-N7S-ADD-001, TASTE SAMV71 runtime targets ATSAMV71Q21 MCU, by using a set of drivers
+# tailored to this platform.
+# \SRS  ETB-RES-20
+def test_TasteSamV71RuntimeTargetsATSAMV71Q21Mcu():
+    pass
+
+##
+# As per chapter 5.1 of ETB-N7S-ADD-001, Hardware Access SEDS implementation targets ATSAMV71Q21 MCU.
+# \SRS  ETB-RES-60
+def test_HwasTargetsATSAMV71Q21Mcu():
+    pass
+
+##
+# As per chapter 5.1 of ETB-N7S-ADD-001, TASTE SAMV71 runtime is based on a FreeRTOS port.
+# \SRS  ETB-DES-10
+def test_TasteSamv71RuntimeBasedOnFreeRtos():
+    pass
+
+##
+# As per chapter 5.1 of ETB-N7S-ADD-001, TASTE SAMV71 runtime uses Sdramc driver to configure SDRAM attached to the MCU.
+# \SRS  ETB-DES-12
+def test_TasteSamv71RuntimeUsesSdram():
+    pass
+
+##
+# TASTE SAMV71 runtime shall use DMA in UART implementation as specified in ETB-N7S-ADD-001 document.
+# As per chapter 5.1 of ETB-N7S-ADD-001, TASTE SAMV71 runtime uses Uart driver to configure UART attached to the MCU.
```

Comment:
No, please refer to the fact, that we use XDMAC in UART communication.

Reply:
changed